release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1516,6 +1516,18 @@ With this release, instead of returning an overflow `servfail` error, the CoreDN
 [id="ocp-4-15-node-bug-fixes"]
 ==== Node
 
+* Due to how the Machine Config Operator (MCO) handles machine configurations for worker pools and custom pools, the MCO might apply an incorrect cgroup version argument for custom pools. As a consequence, nodes in the custom pool might feature an incorrect cgroup kernel argument that causes unpredictable behavior. As a workaround, specify cgroup version kernel arguments for worker and control plane pools only.(link:https://issues.redhat.com/browse/OCPBUGS-19352[*OCPBUGS-19352*])
+
+* Previously, CRI-O was not configuring the cgroup hierarchy correctly to account for the unique way that `crun`` creates cgroups. As a consequence, disabling the CPU quota with a PerformanceProfile did not work. With this fix, using a PerformanceProfile to disable CPU quota works as expected. (link:https://issues.redhat.com/browse/OCPBUGS-20492[*OCPBUGS-20492*])
+
+* Previously, because of a default setting (`container_use_dri_devices, true`), containers were unable to use dri devices. With this fix, containers can use dri devices as expected. (link:https://issues.redhat.com/browse/OCPBUGS-24042[*OCPBUGS-24042*])
+
+* Previously, the kubelet was running with the `unconfined_service_t` SELinux type. As a consequence, all our plugins failed to deploy due to an Selinux denial. With this fix, the kubelet now runs with the `kubelet_exec_t` SELinux type. As a result, plugins deploy as expected. (link:https://issues.redhat.com/browse/OCPBUGS-20022[*OCPBUGS-20022*])
+
+* Previously, the kubelet was running with the `unconfined_service_t` SELinux type. As a consequence, all our plugins failed to deploy due to an Selinux denial. With this fix, the kubelet now runs with the `kubelet_exec_t` SELinux type. As a result, plugins deploy as expected. (link:https://issues.redhat.com/browse/OCPBUGS-20022[*OCPBUGS-20022*])
```

Comment:
This looks to be a duplicated entry for the one just above it

Reply:
Whoops, thanks!